js/app.js

```diff
@@ -97,23 +94,53 @@ function CreateProducts (name) { // object constructor
 productImagesParent.addEventListener ('click', clickHandler);
 
 function clickHandler (event) { // event handler needs to:
-  if (maxClicks > 24) {
+  if (maxClicks > limit) {
     productImagesParent.removeEventListener ('click', clickHandler);
+    renderList();
   }
   // take click
+  console.log(currentGroup.length + 'yaaaay');
```

Comment:
Ha ha, thank you, I don't seem to remember to remove them until the end...maybe I remember then...